pulpcore/cli/common/context.py

```diff
@@ -304,6 +304,7 @@ class PulpEntityContext:
     UPDATE_ID: ClassVar[str]
     DELETE_ID: ClassVar[str]
     NULLABLES: ClassVar[Set[str]] = set()
+    CAPABILITIES: ClassVar[Dict[str, List[PluginRequirement]]] = {}
```

Comment:
I'd like to see a block-comment here describing what capabilities are for, and what one looks like.